ts/src/pro/binance.ts

```diff
@@ -157,6 +157,8 @@ export default class binance extends binanceRest {
                 'ws': {
                     'cost': 5,
                 },
+                // tracks User Data Stream websocket-api subscriptions per market scope
+                'userDataStream': this.createSafeDictionary (),
```

Comment:
@pcriadoperez, what if the connection crashes or the user does `.close()`, can it leave leftovers from the previous connection?

Reply:
Sorry was old code I forgot to delete, the variable is not used.